### PR TITLE
DS additions

### PR DESCRIPTION
**Pull request by thea-m**
added Arabic
@PietroLiuzzo I'm still not sure about the encoding of the translated decorations descriptions (in DSEthiop1, 4 and 8). I left them commented out for now because the schema complains about a missing xml:id.

### CODE REVIEW

**Comment by thea-m** [comment]
@DenisNosnitsin1970, I'm not sure how to solve the conflict in DSEthiop3. The change is having "the" `<origDate>the seventeenth century</origDate>`, which indeed I approved of in the last PR but wonder why now. Do we want "the seventeenth century" in the title line?

File: DayrSuryan/DSEthiop1.xml [hunk 1]
```diff
@@ -561,6 +572,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <note>The legend is written in a crude hand, the vowel marker of <foreign xml:lang="gez">ዝ</foreign> is placed wrongly
                         (at the right leg of the letter, and turned to the right).</note>
                      </decoNote>
+                        
+                  <!--   <decoNote xml:id="drawing" corresp="#d3" xml:lang="ar">
```
Comment (PietroLiuzzo) [inline DayrSuryan/DSEthiop1.xml:576]: the decoration is the same, you need 2 desc one for each xml:lang, but the decoNote should be one, I think

**Review by DenisNosnitsin1970 — APPROVED** [review]

File: DayrSuryan/DSEthiop1.xml (hunk 1 above)
Comment (thea-m) [reply DayrSuryan/DSEthiop1.xml:576]: Same for the locus? For example:
```
    <locus target="#118v"/>
    <locus target="#118v" xml:lang="ar">ورقة ١١٨ ظ</locus>
```

Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop1.xml]: Is it for the Arabic inventory?

Comment (thea-m) [reply DayrSuryan/DSEthiop1.xml]: exactly

**Pull request merged**